app/controllers/api/v1/api_controller.rb

```diff
@@ -43,6 +43,18 @@ def render_objects_with_cache(object, pid)
       end
     end
 
+    # nil.to_i = 0
+    def page
+      @page = params['page'].to_i
+      (@page == 0) ? 1 : @page
+    end
+
+    # max limit is 25
+    def per_page
+      @per_page = params['per_page'].to_i
+      (@per_page == 0 or @per_page > DEFAULT_SEARCH_COUNT) ? DEFAULT_SEARCH_COUNT : @per_page
```

Comment:
Instead of or can we use || ? Let me know if any specific reason to use this.

Reply:
No reason, will change to `||`.